apps/web-app/middleware/auth.global.ts

```diff
@@ -28,10 +28,17 @@ export default defineNuxtRouteMiddleware(async (to) => {
       useAuth().status.value === 'unauthenticated' ||
       useAuth().status.value === 'loading'
     ) {
-      if (toGuest) {
-        return;
+      const { data: verifyAuthStatus } = await useFetch('/api/auth/status', {
+        method: 'GET'
+      });
```

Comment:
Can we double check if using `useFetch` vs `$fetch` is the right choice?

Reply:
if we disable SSR across the whole app, we could use $fetch

Reply:
this is client side only anyway, switching to $fetch